cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -84,28 +85,44 @@ func UpdateAllInstanceNoExternalIP(workflow *daisy.Workflow, noExternalIP bool)
 	if !noExternalIP {
 		return
 	}
-	updateAllInstanceAccessConfig(workflow, func() []*compute.AccessConfig {
-		return []*compute.AccessConfig{}
-	})
-}
-
-func updateAllInstanceAccessConfig(workflow *daisy.Workflow, accessConfigProvider func() []*compute.AccessConfig) {
-	for _, step := range workflow.Steps {
-		if step.IncludeWorkflow != nil {
-			//recurse into included workflow
-			updateAllInstanceAccessConfig(step.IncludeWorkflow.Workflow, accessConfigProvider)
-		}
+	WorkflowStepIterator(workflow, func(step *daisy.Step) {
 		if step.CreateInstances != nil {
 			for _, instance := range *step.CreateInstances {
 				if instance.Instance.NetworkInterfaces == nil {
 					return
 				}
 				for _, networkInterface := range instance.Instance.NetworkInterfaces {
-					networkInterface.AccessConfigs = accessConfigProvider()
+					networkInterface.AccessConfigs = []*compute.AccessConfig{}
 				}
 			}
 		}
-	}
+	})
+}
+
+// UpdateToUEFICompatible updates workflow resources (disks and images) to be UEFI compatible
```

Comment:
It'd be nice for the comment to describe what updates are happening. As it stands, it could be modifying bytes in the image, inserting a new bootloader on the disk, who knows!

Reply:
Updated the description